### PR TITLE
test: XCal events export test

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Helps #5320 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added xcal export test

### CODE REVIEW

File: app/api/helpers/tasks.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def export_event_task(self, email, event_id, settings):
     except Exception as e:
         print(traceback.format_exc())
         result = {'__error': True, 'result': str(e)}
-        logging.info('Error in exporting.. sending email')
+        logging.error('Error in exporting.. sending email')
```
Comment (iamareebjamal) [inline app/api/helpers/tasks.py]: Not an error unless unhandled

**Comment by mrsaicharan1** [comment]
Yes, these are wrong changes from thepast. Have to resolve conflicts

File: tests/all/integration/api/helpers/test_xcal.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+from tests.all.integration.setup_database import Setup
+from tests.all.integration.utils import OpenEventTestCase
+from app import current_app as app
+from app.api.helpers.xcal import XCalExporter
+from xml.etree.ElementTree import fromstring, tostring
+
+from app.factories.event import EventFactoryBasic
+from app.api.helpers.db import save_to_db
+
+class TestLogging(OpenEventTestCase):
```
[the remark this answers is not in the document]
Comment (iamareebjamal) [reply tests/all/integration/api/helpers/test_xcal.py]: TestLogging?

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please review

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please review

**Pull request merged**